bot/.gitignore

```diff
@@ -0,0 +1 @@
+*_freq.txt
```

Comment:
We don't need a second .gitignore so I'm gonna go ahead and delete this for now. If you need to ignore `*_freq.txt` files you can add them to `/.gitignore` instead of `/bot/.gitignore`.

Reply:
Fair enough. The `*_freq.txt` files are to make sure that I'm not producing any common phrases that they can filter out and should be ignored.